pkg/autoops/domain/auto_ops_rule.go

```diff
@@ -74,19 +99,38 @@ func NewAutoOpsRule(
 
 }
 
+func (a *AutoOpsRule) SetStopped() {
+	a.SetAutoOpsStatus(proto.AutoOpsStatus_STOPPED)
+}
+
 func (a *AutoOpsRule) SetDeleted() {
 	a.AutoOpsRule.Deleted = true
 	a.AutoOpsRule.UpdatedAt = time.Now().Unix()
 }
 
+// TODO: Remove this function after auto ops migration.
+// Deprecated
 func (a *AutoOpsRule) SetTriggeredAt() {
 	now := time.Now().Unix()
 	a.AutoOpsRule.TriggeredAt = now
+	a.AutoOpsStatus = proto.AutoOpsStatus_FINISHED
 	a.AutoOpsRule.UpdatedAt = now
 }
 
+func (a *AutoOpsRule) SetFinished() {
+	a.SetAutoOpsStatus(proto.AutoOpsStatus_FINISHED)
+}
+
 func (a *AutoOpsRule) AlreadyTriggered() bool {
```

Comment:
```suggestion
// TODO: Remove this function after auto ops migration.
// Deprecated
func (a *AutoOpsRule) AlreadyTriggered() bool {
```

Reply:
I Fixed it in https://github.com/bucketeer-io/bucketeer/pull/994/commits/552a938717774c5b227dc56f34b6a060acb07444.